README.rst

```diff
@@ -2,20 +2,20 @@
 INSTALL
 #######
 
-To install the project from the development::
+To run the demo project for testing::
 
     $ git clone git://github.com/jojax/django-js-error-hook.git
     $ cd django-js-error-hook
-    $ virtualenv env
-    $ env/bin/python setup.py develop
+    $ python3 -m venv env
```

Comment:
I am not a fan on mixing two versions of Python here.

Reply:
It is not mixing different versions. It is using Python 3 throughout.

This needs to be done because the way you have setup the setup.py file, it automatically installs django 2.0 -- which only runs on python 3.

Reply:
The instructions make the user create a new venv, so there cannot be anything installed in that venv already. I am only switching that venv from being a Python 2 to a Python 3 venv. This is required because the system subsequently tries to install Django 2.0 into the venv -- which won't run in a Python 2 venv.